packages/web/graphql/fragments.ts

```diff
@@ -10,25 +10,25 @@ export const PlayerFragment = gql`
     ethereum_address
     availability_hours
     timezone
-    ColorAspect {
+    color_aspect {
       name
       description
       mask
     }
-    playerType {
+    type {
       description
       id
       imageUrl
       title
     }
-    Player_Skills {
+    skills {
```

Comment:
I'm not sure, you made changes to hasura to reflect this right?
If so there are more references in: 

seed-local-db.js

Reply:
The original change was in hasura, then I manually updated the various references to these types in the backend and frontend code.